src/sentry/models/organizationoption.py

```diff
@@ -59,23 +43,20 @@ def get_all_values(self, organization):
         else:
             organization_id = organization
 
-        if organization_id not in self.__cache:
+        if organization_id not in self._cache:
             cache_key = self._make_key(organization_id)
             result = cache.get(cache_key)
             if result is None:
                 result = self.reload_cache(organization_id)
             else:
-                self.__cache[organization_id] = result
-        return self.__cache.get(organization_id, {})
-
-    def clear_local_cache(self, **kwargs):
-        self.__cache = {}
+                self._cache[organization_id] = result
```

Comment:
It might be a little safer if we properly namespace these local keys now, just since this cache is shared between all `OptionManager` subclasses. Currently we could potentially have clashes between orgs and projects that have the same id. Since we already have a cache key here we could just use that.